data/json/snippets/radio.json

```diff
@@ -13,17 +14,31 @@
+      "Attention folks, this is DJ Dustbowl in with an emergency announcement for anyone that happens to be listening.  I don’t know if it’s just localized to here or is widespread, but crap’s going down outside.  Reality’s breaking, people, just like when this shitshow started.  Stay indoors at all costs, close any curtains and barricade the doors for good measure, and, for the love of God, if you hear voices outside, ignore them!  They’re not human.  We’re going to be going off air and heading below ground while this crapshoot runs its course.  Have faith, people, we’ll all get through this one as well.  Just don’t go Luny on me, and again, don’t, listen, to the voices!",
+      "Ok, ladies and gentlemen, this is DJ Dustbowl, you’re tuned into Dark Days Radio, and if you’re hearing this, then let’s have an amen because it means we all made it out of the storm.  That is, if the storm was a wide-spread thing and not just neatly centered atop my head.  If that’s the case and y’all didn’t catch my broadcast, the bottom line's that reality was breaking, just like when this nonsense started.  Now, I don’t know if it’ll happen again, but let’s use today’s survival tip slot to see what we can learn from it.  Number one, folks, if reality’s shitting the bed, don’t try to fight anything that appears.  If you’re caught out, my advice is to just run for cover, and you’d be best served by snuggling underground.  Failing that, closing all the kirtans and barricading the doors will work in a pinch.  Most importantly, people, if you hear something outside, regardless of what it might be, stuff your ears and ignore it.  You’ll hear things that will make you want to open the front door.  I heard… I opened up.  You don’t want to see what’ll be waiting for you.",
+      "Hey people, it’s Dustbowl again, back gon… on Dark Days Radio.  I’m sorry if I’m slurring my words today, but I’ve eenb hitting the boos real fucking hard.  It’s all okay, I guess.  It’s not like anyone’s listenizng to this trash anyway.  Y’aulll are probably all dead.  God knnows, maybe ty’all are dead because of me as well.\n\nIn his calr.  He was sitdting in his car.  I’m so sorry.  Something had torn it open: hallf of it was missssing when I found it.  It was in better caondition than what used to be …kshssssssk.  He… it, had already reanimated, and …kshhhsk… so many tentacles.  Dear god.  Why cahn’t you just let corpses rot in peace?  Why did you have to turn him into that?  It’s all my fault.  I should havne never asked him to make it over.  DI burned and buried it.  I’m so focking sorry.  Stupid fucking make-believe DJ, pretending to run a radio show and getting people killled becauuse he’s lonely.  I shold stay alone.  I should die alone.\n\nI unterstand now.  I understand why I heard you icn the… portal storm and why you told me to comme out.  Did you want revenge?  I’m so sorry.  I hate myself.  I fucking haatte myself.  I’m sorry, Anarchy.",
```

Comment:
```suggestion
      "Hey people, it’s DJ Dustbowl again, back gon… on Dark Days Radio.  I’m sorry if I’m slurring my words today, but I’ve been hitting the boos real fucking hard.  It’s all okay, I guess.  It’s not like anyone’s listenizng to this trash anyway.  Y’aulll are probably all dead.  God knnows, maybe ty’all are dead because of me as well.\n\nIn his calr.  He was sitdting in his car.  I’m so sorry.  Something had torn it open: hallf of it was missssing when I found it.  It was in better caondition than what used to be …kshssssssk.  He… it, had already reanimated, and …kshhhsk… so many tentacles.  Dear god.  Why cahn’t you just let corpses rot in peace?  Why did you have to turn him into that?  It’s all my fault.  I should havne never asked him to make it over.  DI burned and buried it.  I’m so focking sorry.  Stupid fucking make-believe DJ, pretending to run a radio show and getting people killled becauuse he’s lonely.  I shold stay alone.  I should die alone.\n\nI unterstand now.  I understand why I heard you icn the… portal storm and why you told me to comme out.  Did you want revenge?  I’m so sorry.  I hate myself.  I fucking haatte myself.  I’m sorry, Anarchy.",
```